README.md

```diff
@@ -24,209 +41,46 @@ from discoverx import DX
 dx = DX(locale="US")
 ```
 
-## Maintenance operations
-
-You can run arbitrary SQL operations on multiple tables. 
-For example, to vacuum all the tables in "default" catalog:
-
-```
-dx.from_tables("default.*.*")\
-  .apply_sql("VACUUM {full_table_name}")\
-  .execute()
-```
-
-That will apply the SQL template `VACUUM {full_table_name}` to all tables matched by the pattern `default.*.*`.
-
-The SQL template has the following variables available:
-* `full_table_name` - The table name in format `catalog.schama.table`
-* `table_catalog` - Name of the catalog
-* `table_schema` - Name of the schema
-* `table_name` - Name of the table
-
-You can use the `explain()` command to see the SQL that would be executed.
-
-```
-dx.from_tables("default.*.*")\
-  .apply_sql("VACUUM {full_table_name}")\
-  .explain()
-```
+You can now run operations across multiple tables. 
 
-You can also filter tables that have a specific column name. 
-Eg.
+For example you can select one row from all tables which name contains `sample` in a catalog starting with `dev_` with
```

Comment:
I found the sentence not to easy to follow. ChatGpt thought that the following formulation would be more clear :-)

> As an illustration, consider the scenario where you need to retrieve a single row from various tables within a catalog that begins with "dev_" and includes the term "sample" in their names. To achieve this, the following code block utilizes the dx.from_tables function, which applies an SQL query to extract JSON-formatted data: